### DISCUSSION TITLE
Anyone experiencing iOS 18 issues?

### DISCUSSION THREAD

**Discussion by Gerardv514** [dani-garcia/vaultwarden#5026] [Q&A] [answered]
We are looking to update iPhones to iOS 18 soon. Before doing so id like to gauge whether there are any major issues with the Bitwarden Client apps and iOS 18 along with the Vailtwarden Server backend.

**Comment by fireflyzhu** [comment]
yes, the self-hosted server cannot be log in from ios 18

**Reply by Gerardv514** [reply]
Do you know if that is a bitwarden client issue?

**Reply by kcris** [reply]
yup, can't login using ios 18.0.1 and latest vaultwarden (selfhosted)

**Reply by BlackDex** [reply]
@kcris try the `testing` tagged build as that has some fixes specifically to fix iOS clients.

**Reply by kcris** [reply]
Just tried `vaultwarden/server:testing`. Same error. Thanks

docker logs:

```bash
/--------------------------------------------------------------------\
|                        Starting Vaultwarden                        |
|                           Version 1.28.1                           |
|--------------------------------------------------------------------|
| This is an *unofficial* Bitwarden implementation, DO NOT use the   |
| official channels to report bugs/features, regardless of client.   |
| Send usage/configuration questions or feature requests to:         |
|   https://github.com/dani-garcia/vaultwarden/discussions or        |
|   https://vaultwarden.discourse.group/                             |
| Report suspected bugs/issues in the software itself at:            |
|   https://github.com/dani-garcia/vaultwarden/issues/new            |
\--------------------------------------------------------------------/

[INFO] Using saved config from `data/config.json` for configuration.

[WARNING] The following environment variables are being overriden by the config.json file.
[WARNING] Please use the admin panel to make changes to them:
[WARNING] DOMAIN, SIGNUPS_ALLOWED, ADMIN_TOKEN

[2024-10-17 10:36:53.166][start][INFO] Rocket has launched from http://0.0.0.0:80
[2024-10-17 10:38:49.411][request][INFO] GET /vw/api/config
[2024-10-17 10:38:49.411][response][INFO] (config) GET /vw/api/config => 200 OK
[2024-10-17 10:38:49.431][request][INFO] GET /vw/api/config
[2024-10-17 10:38:49.431][response][INFO] (config) GET /vw/api/config => 200 OK
[2024-10-17 10:39:02.149][request][INFO] GET /vw/api/devices/knowndevice
[2024-10-17 10:39:02.152][response][INFO] (get_known_device) GET /vw/api/devices/knowndevice => 200 OK
[2024-10-17 10:39:15.791][request][INFO] POST /vw/identity/accounts/prelogin
[2024-10-17 10:39:15.793][response][INFO] (prelogin) POST /vw/identity/accounts/prelogin => 200 OK
```

Client (ios 18) says: an error occurred.

**Reply by stefan0xC** [reply]
`Version 1.28.1` is very outdated. Maybe check your `docker version` if you are not able to pull newer images.

**Reply by BlackDex** [reply]
That is also the reason it doesn't work hehe.

**Reply by kcris** [reply]
indeed, got the `Version 1.32.2-65629a99` now and it works. Thank you

**Reply by rjduraocosta** [reply]
Hello to all,

I am new to Vaultwarden. I am selfhosting in Proxmox lxc with the following versions:

  Vaultwarden 1.32.5
  Web-Vault 2024.6.2
  iOS app version: 2024.11.0

On iOS 18 the Bitwarden app issues an error and does not login. Elsewhere everything is working perfectlly.
Anyone noticed anything in the new versions of vaultwarden and bitwarden iOS app?

Thank you in advanced for all the help.

Best regards.

**Reply by BlackDex** [reply]
You say you are new, does that mean no vault entries or just a few? Because if you are not having a lot of vault entries it is strange. How did you created these entries? Which clients?

Do you have the Apple Private Relay feature enabled maybe? What happens if you use cellular data instead of wifi?

Is the reverse proxy you are unsung causing issues? What do those logs mention?

**Reply by rjduraocosta** [reply]
Hello, thank your for your quick reply.

Answering your questions:
I have more or less 20 vault entries. I created all the entries with the Chrome extension of Bitwarden.

Regarding Apple Private Relay I already checked and it is not enabled. With cellular data I ahve the same issue.

The reverse Pproxy is Nginx and I followed this guide in youtube:

https://www.youtube.com/watch?v=4J8s0GiKjBo

Where can I see the logs, please?

Thanks.

**Reply by BlackDex** [reply]
They should either be in the data directory or in the docker logs or somewhere where the kids of the system are placed. I have no clue in your setup. 

I know some nginx setups have security features enabled. Those could cause strange issues. But maybe the logs of nginx could provide more details on that.

**Comment by Gerardv514** [comment, marked as answer]
Updated an iPhone to iOS 18, no issues noticed at the moment.

**Comment by rnarder** [comment]
Hey guys!

Just installed VaultWarden 1.32.5 on my umbrell home, i can access it with BitWarden’s browser extension on Arc on my Mac but still can´t access from the iPhone, i’ve tried using tailscale and Orb but nothing.

Tnx!

**Reply by Gerardv514** [reply]
I’d suggest creating a new discussion, since this thread had to do with if there were any issues with iOS 18. Since there are many already that are running bitwarden clients/Vaultwarden backend on iPhone iOS 18 without issue, your issue is probably more specific to something else.
